src/redux/configureStore.js

```diff
@@ -1,11 +1,11 @@
 import { combineReducers, configureStore } from '@reduxjs/toolkit';
-import bookReducer from './books/Books';
-import categoriesReducer from './categories/Categories';
+import bookReducer from './books/books';
+import categoriesReducer from './categories/categories';
 
 const rootReducer = combineReducers({
-  book: bookReducer,
+  books: bookReducer,
   categories: categoriesReducer,
 });
 
-const store = configureStore(rootReducer);
+const store = configureStore({ reducer: rootReducer });
```

Comment:
- [ ] Same here, There is a problem happing because of this module name, here you name it `Categories.js` with capital `C` and you imported it into your components as `categories` and that's making compiling problems, So please fix that.
Other than that you have done a great job.

Reply:
@Omar-Muhamad Hello, thanks for your review, I don't understand what happened here it's working fine.